src/frontend/tsx/index.tsx

```diff
@@ -1,4 +1,9 @@
 import React from 'react'
+import '../css/buttons.css'
+import '../css/chessground.css'
+import '../css/general.css'
+import '../css/tabs.css'
+import '../css/textboxes.css'
```

Comment:
Any reason for adding these in? Should be taken care of by `index.html`

Reply:
Huh, the webpack docs said they were necessary but both build workflows work without them. Removed.